### PR TITLE
Add Mockery 0.9.3 release to frameworks.yaml

### PR DESCRIPTION
**Pull request by padraic**

### CODE REVIEW

**Comment by padraic** [comment]
Note: CLA should now be recorded.

**Comment by GrahamCampbell** [comment]
You need to re-run the tests and commit the .expects file.

**Comment by padraic** [comment]
@GrahamCampbell Craps! Will do so this afternoon. Thanks for noting!

**Comment by GrahamCampbell** [comment]
Example: https://github.com/facebook/hhvm/commit/3e6a99bfc79de8ddd6c6b9730f175f3580396112.

**Comment by padraic** [comment]
I'm having no joy getting any tests to run...

```sh
padraic@padraic:~/projects/hhvm/hphp/test/frameworks$ hhvm run.php mockery

Warning: No such file or directory in /home/padraic/projects/hhvm/hphp/test/frameworks/run.php on line 367

Updated composer.json found. Updating phpunit binary.

Downloading PHPUnit in order to run tests. There may be an output delay while the download begins.

Warning: symlink(): File exists in /home/padraic/projects/hhvm/hphp/test/frameworks/utils.php on line 283

Fatal error: Uncaught exception 'HH\InvariantException' with message '/home/padraic/projects/hhvm/hphp/test/frameworks/../../hhvm/hhvm does not exist' in :
Stack trace:
#0 /home/padraic/projects/hhvm/hphp/test/frameworks/utils.php(301): HH\invariant_violation()
#1 /home/padraic/projects/hhvm/hphp/test/frameworks/run.php(417): get_runtime_build()
#2 /home/padraic/projects/hhvm/hphp/test/frameworks/run.php(96): get_unit_testing_infra_dependencies()
#3 /home/padraic/projects/hhvm/hphp/test/frameworks/run.php(679): prepare()
#4 /home/padraic/projects/hhvm/hphp/test/frameworks/run.php(686): main()
#5 {main}
```
```
HipHop VM 3.5.0 (rel)
Compiler: tags/HHVM-3.5.0-0-ga90f4733cfa0d8fefdafc7433f758f78cdc75424
Repo schema: b88b6fee8ef78ab638506d212abcbefe74a276ac
```

**Comment by GrahamCampbell** [comment]
You need to be running a nightly version, or a compiled version.

**Comment by padraic** [comment]
That README is going to be PR'd to death by the time this is over ;).

**Comment by steelbrain** [comment]
@padraic If you haven't compiled hhvm from source, then you'll need to ln `/home/padraic/projects/hhvm/hphp/test/frameworks/../../hhvm/hhvm` to `/usr/bin/hhvm`

File: hphp/test/frameworks/results/mockery.expect [hunk 1]
```diff
@@ -727,8 +739,6 @@ Mockery_MockTest::testShouldAllowMockingProtectedMethodReturnsMockInstance
 NamedMockTest::itCreatesANamedMock
 .
 NamedMockTest::itCreatesPassesFurtherArgumentsJustLikeMock
-FATAL
-NamedMockTest::itShouldThrowIfAttemptingToRedefineNamedMock
```
Comment (padraic) [inline hphp/test/frameworks/results/mockery.expect]: Was there intended to be a FATAL in a .expects? Note, I can add this back with a re-record - just not certain of whether they are intended to be retained.

**Comment by padraic** [comment]
@steelbrain Thanks - just compiled a copy.

File: hphp/test/frameworks/results/mockery.expect (hunk 1 above)
Comment (GrahamCampbell) [inline hphp/test/frameworks/results/mockery.expect]: Maybe the hhvm team have changed how recording works since this was last recorded?

Comment (padraic) [inline hphp/test/frameworks/results/mockery.expect]: Could just be a misunderstanding on my part, i.e. the .expects should have the results that will occur at present (incl. FATAL) vs me submitting without the FATAL.

**Pull request closed without merging**